doc/user/versions.md

```diff
@@ -5,13 +5,32 @@ menu: "main"
 weight: 80
 ---
 
-Type | Version | Binary links
------|---------|--------------
-**Latest release** | {{< version >}} | [Linux](https://downloads.mtrlz.dev/materialized-latest-x86_64-unknown-linux-gnu.tar.gz)/[macOS](https://downloads.mtrlz.dev/materialized-latest-x86_64-apple-darwin.tar.gz)
-**Up next (unreleased)** | {{< next_version >}} | Build from [source using `master`](../install/#build-from-source)
+## Stable releases
 
-## Binary links
+Binary tarballs for all stable releases are provided below. Other installation
+options are available for the latest stable release on the [Install
+page](../install).
 
-Version | Links
---------|------
-v0.1.0 | [Linux](https://downloads.mtrlz.dev/materialized-v0.1.0-x86_64-unknown-linux-gnu.tar.gz)/[macOS](https://downloads.mtrlz.dev/materialized-v0.1.0-x86_64-apple-darwin.tar.gz)
+{{< version-list >}}
+
+macOS binary tarballs are expected to work on macOS 10.14 and later. Linux
```

Comment:
Very minor, but can you throw these in a `ul` with a `li` for each OS?

Reply:
Yep, done!